ios-xcframework/XCFrameworkScaffold.xcodeproj/project.pbxproj

```diff
@@ -27,7 +27,6 @@
 		3F12365029F6B23300AF54A4 /* supported-blocks.json in Resources */ = {isa = PBXBuildFile; fileRef = 3F12364F29F6B23300AF54A4 /* supported-blocks.json */; };
 		3F8D0D482A8CB297008A5891 /* assets in Resources */ = {isa = PBXBuildFile; fileRef = 3F8D0D472A8CB297008A5891 /* assets */; };
 		3FA0C5812A1C8C9700600A9A /* App.js in Resources */ = {isa = PBXBuildFile; fileRef = 3FA0C5802A1C8C9700600A9A /* App.js */; };
-		3FC7862B2A53047D00A0DDAF /* hermes.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3FC7862A2A53047D00A0DDAF /* hermes.xcframework */; };
```

Comment:
I'm guessing this removal of the `hermes` XCFramework is related to this change added in `0.72`:

> Do not add "Copy Hermes Framework" script phase to hermes-engine target. ([af6c9e2183](https://github.com/facebook/react-native/commit/af6c9e218305c70e479c75e5ce1a8d633b1e2947) 
([Source](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#ios-specific-28))